### ISSUE TITLE
No error when asking for invalid TensorRT output tensor name

### ISSUE THREAD

**Issue by nieksand** [NVIDIA/tensorrt-inference-server#1025]
**Description**
Specify a non-existing output tensor for a TensorRT model does not trigger a TRTIS error response.

Request headers:
```
{'Content-Type': 'application/octet-stream', 'NV-InferRequest': 'batch_size: 1 input { name: "import/input_1" dims: [224, 224, 3] } output { name: "total_garbage" }'}
```

Response headers:
```
X-Http-Reason: OK
Nv-Inferresponse: model_name: "blah" model_version: 1 batch_size: 1
Nv-Status: code: SUCCESS server_id: "inference:0" request_id: 38
Content-Type: application/octet-stream
Content-Length: 63
```

This appears to be a regression of https://github.com/NVIDIA/tensorrt-inference-server/issues/140.

**TRTIS Information**
I can reproduce this on both 19-10 and 19-12.

**To Reproduce**
Launch TRTIS with TensorRT plan.  Use TRTIS http endpoint and specify an invalid output tensor name in  NV-InferRequest header.

**Expected behavior**
I expect an http 4xx code for the response and an NV-Status not equal to "SUCCESS".

**Comment by deadeyegoodwin** [comment]
Thanks for the report, we will look into it.

**Comment by tanmayv25** [comment]
This issue should be fixed with PR #1048.
